.github/workflows/bench-turbotrace-against-node-nft.yml

```diff
@@ -2,16 +2,14 @@ name: Bench Turbo trace against @vercel/nft
 
 on:
   push:
-    branches: [main, staging]
+    branches: [main, "bench/*"]
```

Comment:
Can't we use the same pattern instead of manually testing `startsWith(github.head_ref, 'bench/')` above?